### PR TITLE
LPS-69501

### PR DESCRIPTION
**Pull request by pyoo47**
Hi Andrea! I've reviewed your pull request and only made on tiny change. I've created a [ticket](https://issues.liferay.com/browse/LRQA-29770) for myself to compensate for the code you had to remove (https://github.com/pyoo47/liferay-portal/commit/14d14ef6732be5b66a637391f443816b5cbd2374). We had included that logic there because Brian specifically wanted source-formatter to run when a change to portal-web is detected.

### CODE REVIEW

**Comment by Ithildir** [comment]
Thank you @pyoo47! So, do you think it's ready to be sent to Brian? Or we should enable the auto-close feature first?

Thank you so much for your help! 😊

File: build-test-batch.xml [hunk 1]
```diff
@@ -2246,6 +2233,53 @@ information. Make sure to commit in all build services results.
 		</run-batch-test>
 	</target>
 
+	<target name="source-format-jdk8">
+		<run-batch-test>
+			<test-action>
+				<if>
+					<contains string="${liferay.releng.branch}" substring="master" />
```
Comment (pyoo47) [inline build-test-batch.xml]: @Ithildir Since this is on the portal side, we are not 100% sure either but both Hashi and I think this should work.

**Comment by pyoo47** [comment]
@lthildir Brian merged my pull request (https://github.com/brianchandotcom/liferay-jenkins-ee/pull/1987) last night. You can send this pull request in to Brian whenever you're ready.

**Comment by Ithildir** [comment]
Sent to https://github.com/brianchandotcom/liferay-portal/pull/45635.

**Pull request closed without merging**